samples/nrf9160/coap_client/src/main.c

```diff
@@ -52,12 +52,10 @@ static int server_resolve(void)
 {
 	int err;
 	struct addrinfo *result;
-	struct addrinfo hints;
-
-	hints.ai_flags = 0;
-	hints.ai_family = AF_INET;
-	hints.ai_socktype = SOCK_DGRAM;
-	hints.ai_protocol = 0;
+	struct addrinfo hints = {
+		.ai_family = AF_INET;
```

Comment:
There should be a comma at the end of line instead of semi-colon for it to compile. Same for the other instances.

Reply:
Right! Should've build before pushing :)